--- origsrc/fbpanel-6.1/data/config/default.in	2010-05-12 13:30:00.000000000 -0500
+++ src/fbpanel-6.1/data/config/default.in	2014-11-26 16:42:26.171447500 -0600
@@ -313,9 +313,9 @@ Plugin {
     }
 }
 
-plugin {
-    type = volume
-}
+#plugin {
+#    type = volume
+#}
 
 #plugin {
 #    type = battery
--- origsrc/fbpanel-6.1/plugins/battery/battery.c	2010-05-05 06:52:59.000000000 -0500
+++ src/fbpanel-6.1/plugins/battery/battery.c	2014-11-26 16:42:26.174947900 -0600
@@ -53,12 +53,15 @@ static gchar *batt_na[] = {
 
 #if defined __linux__
 #include "os_linux.c"
+#elif defined __CYGWIN__
+#include "os_cygwin.c"
 #else
 
-static void
+static gboolean
 battery_update_os(battery_priv *c)
 {
     c->exist = FALSE;
+    RET(FALSE);
 }
 
 #endif
@@ -79,7 +82,7 @@ battery_update(battery_priv *c)
     } else {
         i = batt_na;
         gtk_widget_set_tooltip_markup(((plugin_instance *)c)->pwid,
-            "Runing on AC\nNo battery found");
+            "Running on AC\nNo battery found");
     }
     k->set_icons(&c->meter, i);
     k->set_level(&c->meter, c->level);
--- origsrc/fbpanel-6.1/plugins/battery/os_cygwin.c	1969-12-31 18:00:00.000000000 -0600
+++ src/fbpanel-6.1/plugins/battery/os_cygwin.c	2014-11-26 16:42:26.178448400 -0600
@@ -0,0 +1,19 @@
+#define WIN32_LEAN_AND_MEAN
+#include <X11/Xwindows.h>
+
+static gboolean
+battery_update_os(battery_priv *c)
+{
+    gboolean ret;
+    SYSTEM_POWER_STATUS sps;
+
+    ENTER;
+    c->exist = FALSE;
+    ret = GetSystemPowerStatus(&sps);
+    if (!ret || (sps.BatteryFlag >= 128) || (sps.BatteryLifePercent == 255))
+      RET(FALSE);
+    c->exist = TRUE;
+    c->level = sps.BatteryLifePercent;
+    c->charging = (sps.BatteryFlag & 8);
+    RET(TRUE);
+}
--- origsrc/fbpanel-6.1/plugins/cpu/cpu.c	2010-03-07 02:14:04.000000000 -0600
+++ src/fbpanel-6.1/plugins/cpu/cpu.c	2014-11-26 16:42:26.181948800 -0600
@@ -46,7 +46,7 @@ static chart_class *k;
 
 static void cpu_destructor(plugin_instance *p);
 
-#if defined __linux__
+#if defined __linux__ || defined __CYGWIN__
 static int
 cpu_get_load(cpu_priv *c)
 {
--- origsrc/fbpanel-6.1/plugins/mem/mem.c	2010-04-28 06:39:31.000000000 -0500
+++ src/fbpanel-6.1/plugins/mem/mem.c	2014-11-26 16:42:26.185449300 -0600
@@ -48,7 +48,7 @@ typedef struct
 
 static stats_t stats;
 
-#if defined __linux__
+#if defined __linux__ || defined __CYGWIN__
 #undef MT_ADD
 #define MT_ADD(x) MT_ ## x,
 enum {
--- origsrc/fbpanel-6.1/plugins/volume/volume.c	2010-05-02 04:13:47.000000000 -0500
+++ src/fbpanel-6.1/plugins/volume/volume.c	2014-11-26 16:42:26.188449700 -0600
@@ -9,9 +9,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
-#if defined __linux__
-#include <linux/soundcard.h>
-#endif
+#include <sys/soundcard.h>
 
 //#define DEBUGPRN
 #include "dbg.h"
