--- origsrc/fbpanel-7.0/data/config/Makefile	2015-12-05 00:34:19.000000000 -0600
+++ src/fbpanel-7.0/data/config/Makefile	2016-10-09 23:15:33.772610900 -0500
@@ -2,7 +2,7 @@
 
 TOPDIR := ../..
 
-TEXT = default pager
+TEXT = default pager multiwindow
 all : $(TEXT)
 CLEANLIST += $(TEXT)
 
--- origsrc/fbpanel-7.0/data/config/multiwindow.in	1969-12-31 18:00:00.000000000 -0600
+++ src/fbpanel-7.0/data/config/multiwindow.in	2016-10-09 23:15:45.707064000 -0500
@@ -0,0 +1,58 @@
+# http://fbpanel.sourceforge.net/docs.html#config
+
+# Default settings are commented out
+Global {
+    Edge = top
+    Allign = left
+    WidthType = request
+#   HeightType = pixel
+#   Height = 26
+#   Margin = 0
+#   SetDockType = true
+#   SetPartialStrut = true
+#   Transparent = false
+    TintColor = #363B3B
+#   Alpha = 127
+#   Autohide = false
+    RoundCorners = false
+#   RoundCornersRadius = 7
+#   Layer = above
+    SetLayer = true
+#   MaxElemHeight = 200
+}
+
+plugin {
+    type = menu
+#   expand = false
+#   padding = 0
+    config {
+        icon = X
+        systemmenu {
+        }
+        separator {
+        }
+        item {
+            name = Cygwin Terminal
+            icon = utilities-terminal
+            action = mintty
+        }
+        item {
+            name = Exit Cygwin/X
+            icon = application-exit
+            action = /usr/libexec/fbpanel/xlogout
+        }
+    }
+}
+plugin {
+    type = tray
+    expand = true
+    padding = 2
+}
+plugin {
+    type = icons
+#   expand = false
+#   padding = 0
+    config {
+        DefaultIcon = X
+    }
+}
